scripts/api.ts

```diff
@@ -133,3 +135,22 @@ async function getGovernanceAccountsImpl<TAccount extends GovernanceAccount>(
   }
   return accounts
 }
+
+export const getTokenAccountsByMint = async (
```

Comment:
This function should be in `tokens.tsx`  and return parsed accounts. See `getOwnedTokenAccounts` as an example